app/lib/BaseEditorController.php

```diff
@@ -746,7 +754,7 @@ public function SummaryData($pa_options=null, $pb_ajax_load_displays=FALSE) {
 
 		if (!$this->request->isAjax()) {
 			if((defined('__CA_ENABLE_DEBUG_OUTPUT__') && __CA_ENABLE_DEBUG_OUTPUT__) || (bool)$this->request->config->get('display_template_debugger') || ($this->request->user->getPreference('show_template_debugger') !== 'hide')) {
-				$this->render('../template_test_html.php');
+				$this->renderGeneric('template_test_html.php');
```

Comment:
This template also can't be found in the list_editor summary, so bundled it in with the others.